internal/bminventory/inventory.go

```diff
@@ -6477,26 +6479,58 @@ func isBaremetalBinaryFromAnotherReleaseImageRequired(cpuArchitecture, version s
 		featuresupport.IsFeatureAvailable(models.FeatureSupportLevelIDCLUSTERMANAGEDNETWORKING, version, swag.String(models.ClusterCPUArchitectureArm64))
 }
 
-// updateMonitoredOperators checks the content of the installer configuration and updates the list
-// of monitored operators accordingly. For example, if the installer configuration uses the
-// capabilities mechanism to disable the console then the console operator is removed from the list
-// of monitored operators.
-func (b *bareMetalInventory) updateMonitoredOperators(tx *gorm.DB, cluster *common.Cluster) error {
+// CheckClusterCapabilities sees if a cluster has the capabilities set to the
+// baselineCapability and contains all the capabilities listed in the includes list and excludes
+// any capability listed in the excludes list
+func (b *bareMetalInventory) CheckClusterCapabilities(cluster *common.Cluster, baselineCapability configv1.ClusterVersionCapabilitySet, includes, excludes []configv1.ClusterVersionCapability) (bool, error) {
```

Comment:
The reason I moved it out of the original function and made it public was so it could be used in the BMAC  https://github.com/openshift/assisted-service/pull/7002/files#diff-86415614c6ed40c56c5ca2b5f751a65746ef5c83b61e6f021a4f5d817e4425baR1099

Reply:
Oops, you're right, for some reason I didn't see it at first :facepalm: